src/app/parcels/ActionBar/Actions.tsx

```diff
@@ -93,6 +98,19 @@ const Actions: React.FC<Props> = ({
 }) => {
     const [selectedParcels, setSelectedParcels] = useState<ParcelsTableRow[]>([]);
     const [modalToDisplay, setModalToDisplay] = useState<ActionName | null>(null);
+    const [availableActionsForUserRole, setAvailableActionsForUserRole] = useState<ActionTypes[]>(
```

Comment:
Do we need this to be a useState / useEffect? Is the role likely to change while the user has the page still open - if the role changes it should cause a re-render anyway.

Reply:
Would have thought one could do it without - just defining the actions in terms of the role.